modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,31 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+The registration method with the Salt bundle is the recommended registration method as long as it covers your use case.
+This section explains the advantages and limitations of the current implementation.
+The Salt bundle is provided as the [package]``venv-salt-minion`` that consists of Salt, Python, and the Python modules Salt depends on.
+Using the Salt bundle, in most cases it is no longer needed that the client provides the required Python version or any Python modules.
```

Comment:
I could imagine that someone who reads this will ask themself "In what cases do I need a Python interpreter on the system and in what cases am I good without having one on the client system?"

The next line clarifies this a bit, it says that an interpreter is needed for boostrapping through the WebUI. We also support other bootstrapping mechanisms, isn't the interpreter needed as well?

I wonder if this sentence causes more confusion / uncertainty than it eliminates.

Reply:
Reading it again, I think you are right.  I'll at least temporarily remove it.  Thanks also for all the other clarifications and suggestions!